modules/formatter/src/parsers/ShapeIdParser.scala

```diff
@@ -31,17 +31,17 @@ import cats.parse.Parser
 import cats.parse.Rfc5234.{alpha, digit}
 
 object ShapeIdParser {
-  val underscore: Parser[Underscore] = Parser.charWhere(_ == '_').as(Underscore)
+  val underscore: Parser[Underscore] = Parser.charIn('_').as(Underscore)
   val identifier_chars: Parser[IdentifierChar] =
-    (Parser.charWhere(_ == '_') | digit | alpha).map(IdentifierChar)
+    (Parser.charIn('_') | digit | alpha).map(IdentifierChar)
```

Comment:
`Parser.char('_')`?

Reply:
`Parser.char('_')` returns `Parser[Unit]` but here we keep the char parsed

Reply:
got it!